string type
string feedbackType
string[] servo
string[] setOption
int[] value
